pkg/provider/metadata.go

```diff
@@ -1,21 +1,26 @@
 package provider
 
 import (
+	"context"
 	"encoding/base64"
 	"encoding/xml"
 	"fmt"
 	"github.com/zitadel/logging"
-	"github.com/zitadel/oidc/pkg/op"
-	"github.com/zitadel/saml/pkg/provider/signature"
 	saml_xml "github.com/zitadel/saml/pkg/provider/xml"
 	"github.com/zitadel/saml/pkg/provider/xml/md"
 	"github.com/zitadel/saml/pkg/provider/xml/xenc"
 	"github.com/zitadel/saml/pkg/provider/xml/xml_dsig"
 	"net/http"
+	"time"
+)
+
+const (
+	DefaultValidUntil    time.Duration = 5 * time.Minute
+	DefaultCacheDuration string        = "PT300S"
```

Comment:
PT300S?

Reply:
xs:duration type, https://www.data2type.de/xml-xslt-xslfo/xml-schema/datentypen-referenz/xs-duration, duration how long the receiver of the metadata is allowed to cache it